### DISCUSSION TITLE
Welcome to JamfMigrator Discussions!

### DISCUSSION THREAD

**Discussion by BIG-RAT** [jamfprofessionalservices/JamfMigrator#106] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sbryant82** [comment]
Is it possible to get Jamf Migrator and JamfCPR merged into a single app?

**Reply by BIG-RAT** [reply]
Possible yes, however as it's been in the back of my mind for some time I have to say it's unlikely. Too many other projects keep taking priority.

**Comment by Perseus985** [comment]
Hi, is there a possibility to include an option that allows me to migrate something from a Site to another one?

**Reply by BIG-RAT** [reply]
Are you looking to restrict what objects are available to migrate to those from a particular site. i.e. use a 'site' admin for the source?
Sites on the same server, different servers, or both?

**Reply by BIG-RAT** [reply]
Also, just noticed there is an issue authenticating to get a list of sites. I Need to get that fixed.

**Reply by Perseus985** [reply]
> Are you looking to restrict what objects are available to migrate to those from a particular site. i.e. use a 'site' admin for the source? Sites on the same server, different servers, or both?

Site to Site from the same Server, if possible. I would like to establish a near-production site where the config profiles are pulled from the production site, so that we can test new features and configs and push them to the production site after testing. (We are working in Jamfcloud and not our servers)

**Reply by Perseus985** [reply]
> Also, just noticed there is an issue authenticating to get a list of sites. I Need to get that fixed.

Yeah same. It worked for me when i clicked "Go!" when there was nothing selected in the migrate window, to establish a connection.

**Comment by danlaw777777** [comment]
will this tool overwrite data if its already populated, or will it bypass it entirely?

**Reply by BIG-RAT** [reply]
Wil overrite/update existing objects y default. You have the option to bypass existing items. In settings, select 'Only copy missing items'.
<img width="400" height="451" alt="image" src="https://github.com/user-attachments/assets/0ce22fe2-5e57-469c-be92-bd57aaa89598" />
Of course test and verity you get the desired outcome.

**Comment by danlaw777777** [comment]
Thank you!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by danlaw777777** [comment]
I have a follow up question, will this migrate computers and iOS devices from 1 to another without re enrolling those devices?

Dan Lawrence
From: Dan Lawrence - CW ***@***.***>
Date: Monday, January 5, 2026 at 6:01 PM
To: jamf/Replicator ***@***.***>, jamf/Replicator ***@***.***>
Cc: Comment ***@***.***>
Subject: Re: [jamf/Replicator] Welcome to JamfMigrator Discussions! (Discussion #106)

Thank you!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by BIG-RAT** [comment]
Just moves the record. No changes to client enrollment.
